backend/api_server/requirements.txt

```diff
@@ -6,6 +6,7 @@ gunicorn[gevent] >=20.1.0, <21.0.0
 numba # required for where's my gene
 numpy==1.21.2  # required for where's my gene
 pandas==1.4.3 # required for where's my gene
+pronto==2.4.4
```

Comment:
just checking that does `owlready2` does not provide the descendents extraction functionality we need, in case we can avoid adding this new pkg.